### PR TITLE
domain: unify replayer folder with oom and log

### PR DESCRIPTION
**Pull request by Yisaer**
Signed-off-by: yisaer <disxiaofei@163.com>

### What problem does this PR solve?

Problem Summary:

### What is changed and how it works?

unify replayer folder with oom and log

### Check List

Tests 

- [x] No code

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by hawkingrei — APPROVED** [review]

File: domain/plan_replayer.go [hunk 1]
```diff
@@ -39,7 +40,8 @@ type dumpFileGcChecker struct {
 // GetPlanReplayerDirName returns plan replayer directory path.
 // The path is related to the process id.
 func GetPlanReplayerDirName() string {
-	return filepath.Join(os.TempDir(), "replayer", strconv.Itoa(os.Getpid()))
+	tidbLogDir, _ := filepath.Split(config.GetGlobalConfig().Log.File.Filename)
```
Comment (xhebox) [inline domain/plan_replayer.go:43]: What about `filepath.Dir`?

Comment (Yisaer) [reply domain/plan_replayer.go]: updated

**Review by xhebox — APPROVED** [review]

**Pull request merged**